cmd/minikube/cmd/start.go

```diff
@@ -588,6 +588,7 @@ func selectDriver(oldConfig *cfg.Config) string {
 			out.ErrT(out.Documentation, "Documentation: {{.url}}", out.V{"url": st.Doc})
 		}
 		out.ErrLn("")
+		os.Exit(1)
```

Comment:
I believe we have a wrapper for exit in Out package, please use that

Reply:
Originally the purpose of this PR was FastFail, but I deleted because this is overlaps with # 5840.